doc/source/train/train.rst

```diff
@@ -5,49 +5,142 @@
 Ray Train: Scalable Model Training
 ==================================
 
-.. _`issue on GitHub`: https://github.com/ray-project/ray/issues
+Ray Train is a scalable model training library for distributed training and fine-tuning.
 
-.. tip::
+Ray Train is the simplest way to train large models with large datasets:
 
-    Train is currently in **beta**. Fill out `this short form <https://forms.gle/PXFcJmHwszCwQhqX7>`_ to get involved with Train development!
+* Model training on a single node or multiple nodes, uses the same training code.
+* Ray Train abstracts away the complexities distributed computing for developers.
+* Changing processes and resource allocation with a single line of code.
```

Comment:
I think you're right. Also, I think your original feedback was to put it in bullet points because it was a bit of a wall of text for the landing page. I'm going to move it to the Overview page to see if it works better there. It's missing an intro paragraph.